clang/include/clang/Tooling/CommonOptionsParser.h

```diff
@@ -154,6 +154,39 @@ class ArgumentsAdjustingCompilations : public CompilationDatabase {
   adjustCommands(std::vector<CompileCommand> Commands) const;
 };
 
+#ifdef SYCLomatic_CUSTOMIZATION
+// The MergeCompilationDatabase merges the command line
+// option into the JSONCompilationdatabase if needed. And
+// The tool will skip the heuristic compilation database construct.
```

Comment:
Add comment: this class is used to for compatibility tool to follow the normal process when compilation data is used.